modules/core/shared/src/main/scala/scaladex/core/model/Artifact.scala

```diff
@@ -221,28 +221,59 @@ case class Artifact(
     }
 
   // todo: Add tests for this
-  def scastieURL: String = {
-    val tryBaseUrl = "https://scastie.scala-lang.org/try"
-
-    val targetParam = binaryVersion.platform match {
-      case ScalaJs(_) => Some("t" -> "JS")
-      case _          => None
-    }
-
-    val scalaVersionParam = binaryVersion.language match {
-      case Scala(v) => Some("sv" -> v.toString)
-      case _        => None
-    }
-
-    val params: List[(String, String)] = List(
-      "g" -> groupId.value,
-      "a" -> artifactName.value,
-      "v" -> version.toString
-    ) ++ targetParam ++ scalaVersionParam
-
-    params
-      .map { case (k, v) => s"$k=$v" }
-      .mkString(tryBaseUrl + "?", "&", "")
+  // def scastieURL: String = {
+  //   val tryBaseUrl = "https://scastie.scala-lang.org/try"
+
+  //   val targetParam = binaryVersion.platform match {
+  //     case ScalaJs(_) => Some("t" -> "JS")
+  //     case _          => None
+  //   }
+
+  //   val scalaVersionParam = binaryVersion.language match {
+  //     case Scala(v) => Some("sv" -> v.toString)
+  //     case _        => None
+  //   }
+
+  //   val params: List[(String, String)] = List(
+  //     "g" -> groupId.value,
+  //     "a" -> artifactName.value,
+  //     "v" -> version.toString
+  //   ) ++ targetParam ++ scalaVersionParam
+
+  //   params
+  //     .map { case (k, v) => s"$k=$v" }
+  //     .mkString(tryBaseUrl + "?", "&", "")
+  // }
+
+  // def isSbtPlugin: Boolean = platform match {
+  //   case SbtPlugin(_) => true
+  //   case _            => false
+  // }
+
+  def scastieURL: Option[String] = binaryVersion.platform match {
+    case SbtPlugin(_) | ScalaNative(_) => None
+    case _ =>
+      val tryBaseUrl = "https://scastie.scala-lang.org/try"
+
+      val targetParam = binaryVersion.platform match {
+        case ScalaJs(_) => Some("t" -> "JS")
+        case _          => Some("t" -> "JVM")
+      }
+
+      val scalaVersionParam = binaryVersion.language match {
+        case Scala(v) => Some("sv" -> v.toString)
+        case _        => None
+      }
```

Comment:
I tried the Scastie URL with a missing `sv` param and it did not work. The `sv` param is required.

So if we don't have a Scala version, we cannot build a Scastie URL, and the method should return `None`.